DigitalLearningSolutions.Data/Services/LoginService.cs

```diff
@@ -55,7 +55,10 @@ List<DelegateUser> unverifiedDelegateUsers
             }
 
             var adminUserAssociatedWithDelegate = userDataService.GetAdminUserByUsername(delegateUser.EmailAddress);
-            return adminUserAssociatedWithDelegate?.CentreId == delegateUser.CentreId &&
+            
+            return adminUserAssociatedWithDelegate != null &&
+                   !adminUserAssociatedWithDelegate.IsLocked &&

```

Comment:
I think these two lines can be can be simplified to 
`adminUserAssociatedWithDelegate?.IsLocked == false.`